src/funding/venmo/config.jsx

```diff
@@ -42,7 +42,6 @@ export function getVenmoConfig(): FundingSourceConfig {
         experiment.venmoWebEnabled === false
       ) {
         return {
-          native: true,
           popup: true,
```

Comment:
If device is Mobile & Venmo Web is disabled, then require `popup`? Am I reading that right? From this little snippet it seems like native should be true and popup should be false but I am likely missing context of the bigger picture

I just read your description though, is this saying:

we may still app-switch, but if we can't popup then definitely don't continue

Reply:
@wsbrunson By removing native, it is saying SFVC is ok. We will catch that just prior to app switching. If in SFVC there, then we will fallback to web. Popup relates to WebView and want to keep that there.